sdks/python/apache_beam/io/range_trackers.py

```diff
@@ -24,6 +24,10 @@
 
 from apache_beam.io import iobase
 
+__all__ = ['OffsetRangeTracker', 'GroupedShuffleRangeTracker',
```

Comment:
GroupedShuffleRangeTracker should not be public. (Probably shouldn't even be in this file...)